druid/metadata.csv

```diff
@@ -0,0 +1,114 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+druid.service.health,gauge,,,,"1 if the service is healthy, 0 otherwise",0,druid,health
+druid.query.time,gauge,,millisecond,,Milliseconds taken to complete a query.,-1,druid,qry time
+druid.query.bytes,count,,byte,,Number of bytes returned in query response.,0,druid,qry bytes
+druid.query.node.time,gauge,,millisecond,,Milliseconds taken to query individual historical/realtime processes.,-1,druid,qry node time
```

Comment:
why -1?

Reply:
`druid.query.node.time` is milliseconds taken to query individual historical/realtime processes.

Basically a duration, if it's low, it's good, hence -1.